.env.dist

```
@@ -0,0 +1,128 @@
+_APP_ASSISTANT_OPENAI_API_KEY=STRING
+_APP_BROWSER_HOST=STRING http://appwrite-browser:3000/v1
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)